Tests/SwiftDocCTests/Infrastructure/DocumentationContext/DocumentationContextTests.swift

```diff
@@ -5266,7 +5266,58 @@ let expected = """
             }
         }
     }
-    
+
+    func testContextDiagnosesInsufficientDisambiguationWithCorrectRange() throws {
+        // This test deliberately does not turn on the overloads feature
+        // to ensure the symbol link below does not accidentally resolve correctly.
+        let symbolKindID = SymbolGraph.Symbol.KindIdentifier.allCases.first { !$0.isOverloadableKind }!
```

Comment:
Is this meant to test more than one kind identifier? If not, could the test explicitly specify a know value so that it's clearer which kind identifier is used in the test.

Reply:
It doesn't need to use more than one kind identifier, but i didn't want to depend on a specific symbol kind always being able to be non-overloadable. I can adjust it to use the one that was used when i ran the test (`snippet`) if that would make it clearer, but to me the fact that i pulled an arbitrary non-overloadable symbol kind was the important part.

Reply:
Would it be worth iterating over the kind identifiers (in case there's something odd with one of them)?

Reply:
That's possible. The code i copied to set up the symbols was taken from another test that iterated over all of them, so i could easily take that as well.